src/controllers/crimes_sp.py

```diff
@@ -0,0 +1,67 @@
+from database.db import db
+
+from utils.constants import VALID_CRIMES_SP
+from utils.valid_months import get_all_valid_months
+from utils.amount_crimes import get_cumulative_amounts_of_crimes
+from utils.amount_crimes import get_crimes_per_capita
+from utils.amount_crimes import sort_crimes
+from utils.validators.validate_filters import validade_crime_filters
+
+def get_all_crimes_sp(params, per_capita):
```

Comment:
This method should be generic, a controller named "crimes" that can receive a param and get crimes from 'df' or 'sp' depending on the param.